server/routes/imageRoutes.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const {
+  uploadImage,
+  upload,
+  getImageSrc,
+} = require("../controllers/ImageControllers");
+const { auth } = require("../middlewares/authMiddlewares");
```

Comment:
This middleware doesn't exist on this branch and is crashing the server

Reply:
Yes, I added so that I can make my pull request, when Val made his merge, it will be there and the route will be complete. We can comment these lines out, for now. I will do and commit again so that you can test it.

Reply:
Hey @dproc96 , Since Val merged his branch, I added this line again so that the routes are complete. Since the login/register are not working in frontend right now, it will return not authorized in frontend, but if you test it on postman, and login from there, the routes will work. If Vi merge his branch, the routes will work on frontend as well.

Reply:
Hi again @dproc96 , Just wanted to let you know that the other branch merged to dev too. So I made a pull request and now you can test the routes in the frontend as well. You only need to be logged In and then go to  '/chef' route and test the functionality.